src/Microsoft.AspNet.OData.Shared/Formatter/Deserialization/ODataResourceDeserializer.cs

```diff
@@ -384,6 +384,24 @@ public virtual object CreateResourceInstance(IEdmStructuredTypeReference structu
             }
         }
 
+        /// <summary>
+        /// Deserializes the instance annotations from <paramref name="resourceWrapper"/> into <paramref name="resource"/>.
+        /// </summary>
+        /// <param name="resource">The object into which the structural properties should be read.</param>
+        /// <param name="resourceWrapper">The resource object containing the structural properties.</param>
```

Comment:
>structural properties [](start = 77, length = 21)

"annotations"

Reply:
#Resolved

Reply:
still not fixed in iteration 26.

---
In reply to: [458438101](https://github.com/OData/WebApi/pull/2219#discussion_r458438101) [](ancestors = 458438101)